controllers/util/zk_util.go

```diff
@@ -85,6 +85,10 @@ func GenerateZookeeperCluster(solrCloud *solr.SolrCloud, zkSpec *solr.ZookeeperS
 		zkCluster.Spec.Pod.NodeSelector = zkSpec.ZookeeperPod.NodeSelector
 	}
 
+	if zkSpec.ZookeeperPod.Env != nil {
+		zkCluster.Spec.Pod.Env = zkSpec.ZookeeperPod.Env
```

Comment:
Think you need to add this field to `CopyZookeeperClusterFields`

Reply:
Good catch! Thanks ... always seem to forget about these. I added a basic test for this but only looking at the env ... maybe I should bite the bullet and test all of those fields?

Reply:
Nah, there's already another issue that relates to that. We can tackle it separately and handle all objects at the same time. (They all have the same issue)